locales/en.json

```diff
@@ -512,7 +512,9 @@
     "getStarted": "Get started",
     "unlockApp": "Unlock application",
     "failedToReadKeys": "Failed to read keys",
-    "retryRead": "Want to retry?"
+    "retryRead": "Want to retry?",
+    "decryptError": "Failed to decrypt data",
+    "ignore": "Ignore"
```

Comment:
Should we keep decryptError under "error" key of "WelcomeScreen" since other translations are following this pattern?